docs/changelog-fragments/620.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix SCP transfer crashing with non-existing remote file -- by :user:`Jakuje`
```

Comment:
Could you use the past tense or a non-imperative present simple with the reference to what's changed since the previous release?

Reply:
Something like this works?
```
Fixed crash when downloading non-existent remote file using SCP -- by :user:`Jakuje`
```
(avoiding push to let the CI run)